tensorflow/compiler/xla/service/gpu/gpu_executable.cc

```diff
@@ -476,18 +504,24 @@ StatusOr<ExecutionOutput> GpuExecutable::ExecuteAsyncOnStream(
     absl::optional<HloInputOutputAliasConfig::Alias> alias =
         input_output_alias.GetAliasedParameter(index);
     if (alias) {
-      CHECK_LT(alias->parameter_number, arguments.size());
-      ExecutionInput& input = arguments[alias->parameter_number];
-      MaybeOwningDeviceMemory* maybe_owning_memory =
-          input.MutableBuffer(alias->parameter_index);
-      if (alias->must_alias() && !maybe_owning_memory->HasOwnership()) {
+      MaybeOwningDeviceMemory* maybe_owning_memory = nullptr;
```

Comment:
Optional: again I think initialization with a lambda would be cleaner:

```
maybe_owning_memory = [&] {
    if (...) { 
       return nullptr;
    }
    return ...
}
```

Reply:
done